web/src/components/MapView.tsx

```diff
@@ -47,18 +51,18 @@ const geolocateControlStyle: React.CSSProperties = {
 
 type MapViewProps = {
   bboxViewport: MapViewViewport
-  featureCollection: PoiFeatureCollection
+  featureCollection: PoiFeatureCollection | null
```

Comment:
when is the `featureCollection` null? Couldn't it just be empty?

Reply:
its not an array so it can't be empty. Indeed in the `PoisPage` the `featureLocations` can be empty if there are no pois. These array gonna be embedded in the collection before passing to MapsView